### DISCUSSION TITLE
R abort when fitting the model

### DISCUSSION THREAD

**Discussion by MahmoudAliAfifi** [nlmixr2/nlmixr2#59] [Q&A]
R abort when fitting the model and symengine wrote in the console. I launched the library symengine but the problem is still exist.
Could u help me please?

**Comment by MahmoudAliAfifi** [comment]
![image](https://user-images.githubusercontent.com/115409927/194760840-5d9462e6-8c7f-42db-9758-584cab9245a7.png)

**Reply by mattfidler** [reply]
Hi,

Thank you for reporting this in the discussions so others can see the answer.

Currently CRAN has asked for an updated `dparser`.  We complied, and are working to submit another version of `rxode2` and `nlmixr2`, which should fix this long-term. In the short-term, the currently CRAN bundled binaries of `rxode2` and `nlmixr2est` are incompatible with `dparser` (which will be fixed on windows CRAN but likely not on mac CRAN).  

To fix this you should be able to run the following:

```r
install.packages("rxode2", type="source")
install.packages("nlmixr2est", type="source)
```

This should fix the binary incompatibility.

This tends to occur on a mac frequently, though window builds are built every so often so this doens't happen as much on windwows.
